### PR TITLE
MINT-1656 Implement test rpc login feature

### PR DESCRIPTION
**Pull request by ex1st0r**

### CODE REVIEW

File: packages/login-ui/components/LoginForm/LoginForm.js [hunk 1]
```diff
@@ -6,13 +6,17 @@
...
+import classnames from 'classnames'
 import { Link } from 'react-router'
 import { reduxForm, Field } from 'redux-form/immutable'
 import { TextField } from 'redux-form-material-ui'
 import { connect } from 'react-redux'
 import { Translate } from 'react-redux-i18n'
 import Button from 'components/common/ui/Button/Button'
 import UserRow from 'components/common/ui/UserRow/UserRow'
+import LoginLocal from '../LoginLocal/LoginLocal'
+import NetworkSelector from '@chronobank/login-ui/components/NetworkSelector/NetworkSelector'
```
Comment (ozalexo) [inline packages/login-ui/components/LoginForm/LoginForm.js]: Do not use import from itself. Please replace it to relative path.

File: packages/login/network/settings.js [hunk 2]
```diff
@@ -268,13 +268,13 @@ export const getNetworksByProvider = (providerId, withLocal = false) => {
   }
 }
 
-export const getNetworksWithProviders = (withLocal = false) => {
+export const getNetworksWithProviders = (providers = [], withLocal = false) => {
   let networks = []
+  let providersArray = [...providers]
```
Comment (ozalexo) [inline packages/login/network/settings.js]: Please replace let to const.

And why not just `const providersArray = providers`?

File: packages/login/redux/network/actions.js [hunk 3]
```diff
@@ -19,14 +19,20 @@ import bip39 from 'bip39'
 import Accounts from 'web3-eth-accounts'
 import { login } from '@chronobank/core/redux/session/actions'
 import { stopSubmit, SubmissionError } from 'redux-form'
-import { push } from 'react-router-redux'
+import { push, goBack } from 'react-router-redux'
```
Comment (ozalexo) [inline packages/login/redux/network/actions.js]: **[MAJOR]**

@chronobank/login must not depend on web-related things.
Please replace this import to import from appropriate @chronobank/core-dependencies
(in ChronoMint-RN we will provide our own empty push and goBack).

We can't even build with current implementation.

File: packages/core/redux/persistAccount/utils.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import {
   AccountProfileModel,
 } from '../../models/wallet/persistAccount'
 
-import profileImgJPG from 'assets/img/profile-photo-1.jpg'
+import profileImgJPG from 'platform/assets/img/profile-photo-1.jpg'
```
Comment (ozalexo) [inline packages/core/redux/persistAccount/utils.js]: Please move picture into assets folder and reedport it in platform folder

File: packages/core/redux/persistAccount/actions.js [hunk 5]
```diff
@@ -117,10 +116,14 @@ export const createAccount = ({ name, password, privateKey, mnemonic, numberOfAc
 
   let wallet, hex = privateKey || bip39.mnemonicToSeedHex(mnemonic) || ''
 
-  let host = networkService.getProviderSettings().url
+  let settings = networkService.getProviderSettings()
+  let host = settings.url
```
Comment (ozalexo) [inline packages/core/redux/persistAccount/actions.js]: let -> const at lines 119 and 120

File: packages/core/redux/persistAccount/utils.js [hunk 6]
```diff
@@ -1,10 +1,12 @@
+import {
+  profileImgJPG
+} from 'platform/assets'
```
Comment (ozalexo) [inline packages/core/redux/persistAccount/utils.js]: it must be `from '@chronobank/core-dependencies'`

File: src/platform/assets/index.js [hunk 7]
```diff
@@ -0,0 +1 @@
+export profileImgJPG from 'platform/assets/img/profile-photo-1.jpg'
```
Comment (ozalexo) [inline src/platform/assets/index.js]: I suppose that file profile-photo-1.jpg must be placed at src/assets/img folder

File: packages/login-ui/components/LoginLocal/LoginLocal.scss [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+.account-item {
+  padding: 20px 12px 20px 30px;
+  font-size: 14px;
+  font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  min-height: 82px;
+  border-bottom: 1px solid #E5E5E5;
```
Comment (wAISw) [inline packages/login-ui/components/LoginLocal/LoginLocal.scss]: Use styles variable

File: packages/login-ui/components/LoginLocal/LoginLocal.scss [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+  background: #fff;
```
Comment (wAISw) [inline packages/login-ui/components/LoginLocal/LoginLocal.scss]: Use styles variable

File: packages/login-ui/components/LoginLocal/LoginLocal.scss [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+
+  &:hover, &.providerItemActive {
+    background: #F2F2F2;
```
Comment (wAISw) [inline packages/login-ui/components/LoginLocal/LoginLocal.scss]: Use styles variable

File: packages/login-ui/components/LoginLocal/LoginLocal.scss [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+
+.account-item-content {
+  color: #333;
```
Comment (wAISw) [inline packages/login-ui/components/LoginLocal/LoginLocal.scss]: Use styles variable

File: packages/login-ui/components/LoginLocal/LoginLocal.scss [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+  text-overflow: ellipsis;
+  min-width: 0;
+  word-wrap: break-word;
+  margin-right: 10px;
+}
+
+.account-item-icon {
+  display: flex;
+  align-items: center;
+  color: #BFBFBF;
```
Comment (wAISw) [inline packages/login-ui/components/LoginLocal/LoginLocal.scss]: Use styles variable

**Pull request merged**